shells/pipes-shell/node/node.js

```diff
@@ -27,17 +30,30 @@ import {DeviceApiFactory} from '../device.js';
 //
 // results returned via `DeviceClient.foundSuggestions(arcid, json)` (if it exists)
 
+//const storage = `pouchdb://local/arcs/`;
+const storage = `volatile://`;
+const version = `version: mar-26`;
+
+console.log(`${version} -- ${storage}`);
+
+process.on('uncaughtException', (err) => {
+  console.error('uncaughtException:', err);
+  process.exit(1); //mandatory (as per the Node docs)
+});
+
 (async () => {
-  console.log(`version: feb-27.0`);
-  global.ShellApi = await DeviceApiFactory(`volatile://`, global.DeviceClient);
   // configure arcs environment
   Utils.init(paths.root, paths.map);
+  // configure ShellApi (DeviceClient is bound in by outer process, otherwise undefined)
+  global.ShellApi = await DeviceApiFactory(storage, manifest, global.DeviceClient);
 })();
 
-// test it
+setTimeout(() => test_on_start(), 1000);
```

Comment:
Maybe we want this commented out for a build @cromwellian or @yuangu-google will use. They could do themselves when producing build artifact, either way works.

Reply:
Yep, I meant to disable that, oops.